src/server/prod.ts

```diff
@@ -2,10 +2,26 @@ import compression from 'compression';
 import history from 'connect-history-api-fallback';
 import express from 'express';
 import http from 'http';
+import minimist from 'minimist';
 import favicon from 'serve-favicon';
+import sio from 'socket.io';
+
+import { WebSocketProxyUDPServer } from './network/network';
+import { WebSocketServer } from './network/web_socket_server';
+
+const args = minimist(process.argv.slice(2));
```

Comment:
I would change the arguments to be more like `'NUbots:239.226.152.162:12345'` or `'Blue Team:239.226.152.162:12345'` and just give a bunch of those instead of hard coded values